libs/motor/inc/public/tfc/motors/errors.hpp

```diff
@@ -0,0 +1,38 @@
+#pragma once
+#include <system_error>
+namespace tfc::motor {
+
+// Predefined motor errors
+
+// TODO: think about this some more.
+// how can we allow individual vfds or
+// servo controllers to express their own
+// error codes.
+
+// I don't really believe
+// that control level code cares
+// 'what' is wrong with each motor.,
+// just that it does not turn.
+// So this could maybe be exposed in
+// a dbus api for the hmi and scada or
+// inside some alarm manager.
+namespace errors {
```

Comment:
```suggestion
namespace error {
```
I rather like singular expression here compared to plural but your choice